test/model-definition.test.js

```diff
@@ -375,13 +375,10 @@ describe('ModelDefinition class', function() {
     });
   });
 
-  it('should report deprecation warning for property names containing dot', function() {
-    var message = 'deprecation not reported';
-    process.once('deprecation', function(err) { message = err.message; });
-
-    memory.createModel('Dotted', { 'dot.name': String });
-
-    message.should.match(/Dotted.*dot\.name/);
+  it('should throw error for property names containing dot', function() {
+    (function() { memory.createModel('Dotted', { 'dot.name': String }); })
+      .should
+      .throw('Property names containing dot(s) are not supported: dot.name');
```

Comment:
Please make the assertion less specific, so that we don't have to change this test whenever we improve the error message.

Things that are important to verify: the error message contains the property name, the model name, and it's referring  to the error about dot names.

```js
.should.throw(/dot(s).*dot\.name.*Dotted/);
```